src/Groups/GAPGroups.jl

```diff
@@ -2217,22 +2217,23 @@ function _map_word_syllable(vi::Int, genimgs::Vector, genimgs_inv::Vector)
   vi = -vi
   @assert vi <= length(genimgs)
   isassigned(genimgs_inv, vi) && return genimgs_inv[vi]
-  res = inv(genimgs[vi])
-  genimgs_inv[vi] = res
-  return res
+  genimgs_inv[vi] = one(genimgs[1])
+  genimgs_inv[vi] = inv!(genimgs_inv[vi], genimgs[vi])
+  return genimgs_inv[vi]
 end
 
 function _map_word_syllable(vi::Pair{Int, Int}, genimgs::Vector, genimgs_inv::Vector)
   x = vi[1]
   @assert (x > 0 && x <= length(genimgs))
   e = vi[2]
-  e > 1 && return genimgs[x]^e
   e == 1 && return genimgs[x]
-  isassigned(genimgs_inv, x) && return genimgs_inv[x]^-e
-  res = inv(genimgs[x])
-  genimgs_inv[x] = res
-  e == -1 && return res
-  return res^-e
+  res = one(genimgs[1])
```

Comment:
same thing happens here. if you just allocate the `res` and modify it exactly once, there is no benefit over just calling the non-modifying function instead and not allocating `res` after all